servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -424,18 +423,16 @@ public void moveTempBlockTest() throws Exception {
     mBlockStore.moveBlock(SESSION_ID1, TEMP_BLOCK_ID, mTestDir2.toBlockStoreLocation());
   }
 
-  // TACHYON-825
-  @Ignore
   @Test
-  public void moveBlockToTheLocationWithExistingIdTest() throws Exception {
+  public void cacheSameBlockInDifferentTiersTest() throws Exception {
     mThrown.expect(BlockAlreadyExistsException.class);
-    mThrown.expectMessage("Failed to add BlockMeta: blockId " + BLOCK_ID1 + " exists");
+    mThrown.expectMessage(ExceptionMessage.ADD_EXISTING_BLOCK.getMessage(BLOCK_ID1,
+        FIRST_TIER_ALIAS));
```

Comment:
better to put line 429 and 430 before the second `TieredBlockStoreTestUtils.cache`, so we know explicitly the second `cache` is the expected call to trigger this exception.